neuro_data/static_images/dataset_config.py

```diff
@@ -0,0 +1,533 @@
+        linear_monitor                     : tinyint                                                  # whether to compute linear monitor
+        """
+        content = [
+            dict(stats_tier="train", stats_per_input=1, linear_monitor=0),
+        ]
+
+        @staticmethod
+        def run_stats_resp(data, ix, axis=0):
+            ret = {}
+            data = data[ix]
+            ret["all"] = dict(
+                mean=data.mean(axis=axis).astype(np.float32),
+                std=data.std(axis=axis, ddof=1).astype(np.float32),
+                min=data.min(axis=axis).astype(np.float32),
+                max=data.max(axis=axis).astype(np.float32),
+                median=np.median(data, axis=axis).astype(np.float32),
+            )
+            ret["stimulus.Frame"] = ret["all"]
+            return ret
+
+        @staticmethod
+        def run_stats_input(data, ix, per_input=False):
+            ret = {}
+            data = data[ix]
+            ret["all"] = dict(
+                mean=data.mean(axis=(-1, -2)).mean().astype(np.float32)
+                if per_input
+                else data.mean().astype(np.float32),
+                std=data.std(axis=(-1, -2)).mean().astype(np.float32)
+                if per_input
+                else data.std(ddof=1).astype(np.float32),
+                min=data.min().astype(np.float32),
+                max=data.max().astype(np.float32),
+                median=np.median(data).astype(np.float32),
+            )
+            ret["stimulus.Frame"] = ret["all"]
+            return ret
+
+        def stats(
+            self, dynamic_scan, condition_hashes, trial_idx, images, responses, tiers
+        ):
+
+            key = self.fetch1()
+            # check if the method is eligible for condition_hashes requested
+            assert (
+                (
+                    stimulus.Condition
+                    & "condition_hash in {}".format(tuple(condition_hashes))
+                ).fetch("stimulus_type")
+                == "stimulus.Frame"
+            ).all(), "StatsConfig.NeuroStaticNoBehFrame is only implemented for stimulus.Frame"
+            image_classes = (
+                stimulus.Condition * stimulus.Frame
+                & "condition_hash in {}".format(tuple(condition_hashes))
+            ).fetch("image_class")
+            assert set(image_classes) <= set(
+                FF_CLASSES
+            ), "StatsConfig.NeuroStaticNoBehFrame is only implemented for full-field stimulus"
+
+            # reshape inputs
+            images = np.stack(images)
+            if len(images.shape) == 3:
+                log.info("Adding channel dimension")
+                images = images[:, None, ...]
+            elif len(images.shape) == 4:
+                images = images.transpose(0, 3, 1, 2)
+
+            # gamma correction
+            if key["linear_monitor"]:
```

Comment:
this should be key['gamma']. And we probably want to pend this part until we agree on a luminance conversion table that everyone uses. This table multi_mei.ClosestCalibration is very outdated.